### PR TITLE
#4 feat(GithubTopLanguageBadge)

### PR DESCRIPTION
**Pull request by KwanJunWen**
### Added

- `GithubTopLanguageBadge` which display a badge with the top language used for a GitHub repository.

### Changed

- `BadgesFromPkg` to include `GithubTopLanguageBadge` by default.
- End to end tests which includes `GithubTopLanguageBadge`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 [![open issues](https://img.shields.io/github/issues-raw/dbartholomae/jsx-readme.svg)](https://github.com/dbartholomae/jsx-readme/issues)
 [![dependency status](https://david-dm.org/dbartholomae/jsx-readme.svg?theme=shields.io)](https://david-dm.org/dbartholomae/jsx-readme)
 [![devDependency status](https://david-dm.org/dbartholomae/jsx-readme/dev-status.svg)](https://david-dm.org/dbartholomae/jsx-readme?type=dev)
+[![language](https://img.shields.io/github/languages/top/dbartholomae/jsx-readme)](https://github.com/dbartholomae/jsx-readme)
```
Comment (dbartholomae) [inline README.md:9]: Instead of `language` it should be `GitHub top language`. For the link I like the choice of linking to the repo :)

**Review by dbartholomae — CHANGES_REQUESTED** [review]
Looks mostly great, just one small change! :)

**Comment by dbartholomae** [comment]
One more thing - could you squash all commits into one? The commit messages are used for the changelog and the current level of detail would not be of interest for a user of the library

File: README.md (hunk 1 above)
Comment (KwanJunWen) [reply README.md:9]: Hi, isn't the link already linking to the Github repository ?

Comment (dbartholomae) [reply README.md:9]: Yes, that's what I meant: It was not specified in the ticket, but I like how it is done in the PR.

**Review by dbartholomae — APPROVED** [review]
lgtm

**Pull request merged**